share/revng/test/tests/analysis/CollectCFG/x86_64/switch-jump-table.S

```diff
@@ -3,24 +3,45 @@
 #
 
     .intel_syntax noprefix
-    .type _start, @function
-    .globl _start
-_start:
-    cmp    rax,0x3
+    .type myfunc, @function
+    .globl myfunc
+myfunc:
+    mov    rax, QWORD PTR [rax]
+    cmp    rax, 0x5
```

Comment:
Add a detailed comment about why we need this and under what conditions it can be dropped (i.e., `AdvancedValueInfo` does not play well with tracking arguments).